example/package.json

```diff
@@ -3,19 +3,26 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "start": "npm run build && npm run serve",
     "dev": "cross-env DEBUG=vite-plugin-pwa:* vite",
     "build": "cross-env DEBUG=vite-plugin-pwa:* vite build",
-    "serve": "vite preview"
+    "run-build": "cross-env DEBUG=vite-plugin-pwa:* BASE_URL=/ SOURCE_MAP=true vite build",
+    "serve": "serve dist"
   },
   "dependencies": {
-    "vue": "^3.0.6"
+    "vue": "^3.0.7"
   },
   "devDependencies": {
+    "@rollup/plugin-replace": "^2.4.1",
+    "@types/node-static": "^0.7.6",
```

Comment:
Is `@types/node-static` dependency necessary once added `https-localhost`?

Reply:
Yeah, we can remove it